main.go

```diff
@@ -10,10 +10,11 @@ import (
 	_ "embed"
 	"flag"
 	"fmt"
-	"github.com/vmware/cluster-api-provider-cloud-director/release"
 	"os"
 	"time"
 
+	"github.com/vmware/cluster-api-provider-cloud-director/release"
```

Comment:
I think this is not needed

Reply:
This is basically change from go-imports which arranges the import. It is based on the best practices where it organises the imports in below order 

1. system packages 

2. third party packages

3. local packages 

That is why you see this change. I would suggest adopting this in the project through golang linting.